common/model-views.cpp

```diff
@@ -4215,7 +4215,7 @@ namespace rs2
                     fps = s.second.profile.fps();
             }
             auto curr_frame = p.get_position();
-            uint64_t step = fps ? uint64_t(1000.0 / (float)fps * 1e6) : 1e6;
+            uint64_t step = fps ? uint64_t(1000.0 / (float)fps * 1e6) : 1000000ULL;
```

Comment:
CppCheck?

Reply:
yep

Reply:
well, actually, nope. I just saw the warning when compiling it...